### ISSUE TITLE
Cli interface

### ISSUE THREAD

**Issue by yxliang01** [popomore/chmod#1]
I think it would be great if cli interface can be implemented for this package. You then can implement cross-platform chmod tool like https://www.npmjs.com/package/rimraf.

**Comment by popomore** [comment]
It seems good, but I don't know anyone use this package on Windows.

**Comment by yxliang01** [comment]
I would like to use this on windows if this is available via CLI interface 🎃

**Comment by demurgos** [comment]
I agree that this is a good idea, even if this is a no-op on Windows. It would allow to use it in npm scripts without worrying about the OS.
